console-src/package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "backstage",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@kontist/client": "^0.0.11",
+    "@types/classnames": "^2.2.9",
+    "@types/jest": "24.0.19",
+    "@types/node": "12.11.1",
+    "@types/react": "16.9.9",
+    "@types/react-dom": "16.9.2",
+    "@types/react-modal": "^3.10.0",
+    "@types/react-router-dom": "^5.1.0",
+    "@types/styled-components": "^4.1.19",
+    "classnames": "^2.2.6",
+    "history": "^4.10.1",
+    "react": "^16.10.2",
+    "react-dom": "^16.10.2",
+    "react-modal": "^3.10.1",
+    "react-router-dom": "^5.1.2",
+    "react-scripts": "3.2.0",
+    "styled-components": "^4.4.0",
+    "typescript": "3.6.4"
+  },
+  "scripts": {
+    "start": "react-scripts start",
+    "build": "react-scripts build",
+    "postbuild": "node ./move-dist.js",
+    "test": "react-scripts test",
+    "lint": "eslint --ext .js,.ts,.tsx . --max-warnings 35",
+    "eject": "react-scripts eject"
+  },
+  "eslintConfig": {
+    "extends": "react-app"
+  },
+  "browserslist": {
+    "production": [
+      ">0.2%",
+      "not dead",
+      "not op_mini all"
+    ],
+    "development": [
+      "last 1 chrome version",
+      "last 1 firefox version",
+      "last 1 safari version"
+    ]
+  },
+  "devDependencies": {
+    "eslint-plugin-prettier": "^3.1.1",
+    "prettier": "^1.18.2",
+    "rimraf": "^3.0.0"
+  }
```

Comment:
Do we need to setup a `homepage` in package.json as we do not run the site in root but `/console`? see https://create-react-app.dev/docs/deployment/#building-for-relative-paths

Reply:
this is probably a good idea yes, I have also updated the setup of react router to account for this subdirectory:
e38d72c8ceaa12174d148503064af0a13b8540a3 and 5ddd4f96250c4ea5c5a9af93e9bcc5c2471aa234